homeassistant/components/bmw_connected_drive/lock.py

```diff
@@ -59,9 +69,18 @@ def name(self):
     def device_state_attributes(self):
         """Return the state attributes of the lock."""
         vehicle_state = self._vehicle.state
+        if self.door_lock_state_available:
+            door_lock_state = vehicle_state.door_lock_state.value
+            last_update_reason = vehicle_state.last_update_reason
+        else:
+            door_lock_state = STATE_UNKNOWN
```

Comment:
Our standard is to not include the item at all in the state attributes of the value is unknown.

Reply:
Done